docs/exports.md

```diff
@@ -4,16 +4,143 @@ nav_order: 7
 
 # Exporting to CDS
 
-All data collected by the application needs to be exported to the Consolidated Data Store (CDS) which is a data warehouse based on MS SQL running in the DAP (Data Analytics Platform).
+All data collected by the application needs to be exported to the **Consolidated Data Store (CDS)**, which is a data warehouse based on **MS SQL** running in the **DAP (Data Analytics Platform)**.
+
+This is done via XML exports saved in an **S3 bucket**.  
+Currently, we export the following:
+
+- **Lettings logs**
+- **Users**
+- **Organisations**
 
-This is done via XML exports saved in an S3 bucket.
-We currently export lettings logs, users and organisations.
 The data mapping for these exports can be found in:
 
-- Lettings logs `app/services/exports/lettings_log_export_service.rb`
-- Organisations `app/services/exports/organisation_export_service.rb`
-- Users `app/services/exports/user_export_service.rb`
+- **Lettings logs**: `app/services/exports/lettings_log_export_service.rb`
+- **Organisations**: `app/services/exports/organisation_export_service.rb`
+- **Users**: `app/services/exports/user_export_service.rb`
+
+<!-- Add sales log when sales logs export is added -->
+
+Lettings logs exports are year-specific, so at any given time, there may be records exported for three different years, creating three distinct collections: **previous**, **current**, and **next**. While this is technically possible, it is very unlikely to happen in a production environment unless records are updated manually.
+
+Typically, **one lettings logs collection (current)** is exported most of the time, and **two collections (current and previous)** are exported during the crossover period. Export objects are still created for all collections, but they are marked as **empty** when applicable, and no files are generated for these empty exports.
+
+**Users** and **organisations** are not year-specific exports, so there will never be more than one collection for each of these in a single partial export.
 
 Initially the application database field names and field types were chosen to match the existing CDS data as closely as possible to minimise the amount of transformation needed. This has led to a less than optimal data model though and increasingly we should look to transform at the mapping layer where beneficial for our application.
 
-We have a cron job triggering the export service daily at 5am.
+A **cron job** triggers the export service daily at **5 a.m.**.
```

Comment:
Is this 5am UTC or 5am in the timezone that includes London? (I.e. what happens during daylight savings time?)

I suspect this will never matter, but might be useful to write down just in case something strange happens at any point.